mono/metadata/icall.c

```diff
@@ -9844,9 +9844,15 @@ ves_icall_System_Net_NetworkInformation_LinuxNetworkChange_CloseNLSocket (gpoint
 
 #include "metadata/icall-def.h"
 
-#undef HANDLES
-#undef HANDLES_REUSE_WRAPPER
-#undef ICALL_TYPE
-#undef ICALL
-#undef NOHANDLES
-#undef MONO_HANDLE_REGISTER_ICALL
```

Comment:
This seems like an unintended change?

Reply:
Yes, this is the same as above, I'll review the merge I did.

Reply:
Ok, I've fixed the changes. This diff looks much better now.